Gemfile

```diff
@@ -56,6 +56,7 @@ gem "inifile",                        "~>3.0",         :require => false
 gem "jbuilder",                       "~>2.5.0" # For the REST API
 gem "manageiq-api-client",            "~>0.1.0",       :require => false
 gem "mime-types",                     "~>2.6.1",       :require => "mime/types/columnar"
+gem "money-rails",                    "~>1"
```

Comment:
yeah, @sergio-ocon is there a single entry point to run the initializer and require money-rails?

I don't see any calls to MoneyRails in here outside of the initializer.  When is this needed?